pytest/tests/sanity/replay_chain_from_archive.py

```diff
@@ -0,0 +1,143 @@
+#!/usr/bin/env python3
+# Spins up 4 validating nodes, 1 non-validating RPC node, and 1 non-validating archival node.
```

Comment:
Any particular reason to have this layout of nodes? If I were to change number of validators to 2 does the test become weaker?